dagger-core/src/main/java/io/odpf/dagger/core/config/ConfigurationProviderFactory.java

```diff
@@ -19,7 +19,7 @@
     public ConfigurationProviderFactory(String[] args) {
 
         this.args = args;
-        System.out.println("Arguments are : ");
+        System.out.println("Arguments are: ");
```

Comment:
why do we have sysout? lets use proper logging?

Reply:
It was there earlier, but yes - can add a logger here.